week1/week1.py

```diff
@@ -0,0 +1,58 @@
+##########Global Variables##########
+
+###########Primitive Data Types##########
+#python isn't a typed language like C++ or Java
+#This is a comment!
+
+'''
+This is a block comment!
+You can write more than
+one line here.
+'''
+myName = "George Burdell" #string
+middleInitial = 'P' #char
+yearsInRoboJackets = 3 #int
+myBirthday = 8.11 #float
+isRoboJacketsCool = True #boolean
+
+###########Lists##########
+gpas = [1.0, 2.6, 4.0, 3.6]
+membersInRoboJackets = []
+
+def exampleFunction():
```

Comment:
the actual python method `__main__`? I would personally stick to just calling it `main` for readability, since it dosen't matter in our use case (I think).